src/ray/gcs/pubsub/test/gcs_pub_sub_test.cc

```diff
@@ -129,6 +131,29 @@ TEST_F(GcsPubSubTest, TestPubSubApi) {
   WaitPendingDone(all_result, 3);
 }
 
+TEST_F(GcsPubSubTest, TestManyPubsub) {
+  std::string channel("channel");
+  std::string id("id");
+  std::string data("data");
+  std::vector<std::pair<std::string, std::string>> all_result;
+  SubscribeAll(channel, all_result);
+  // Test many concurrent subscribes and unsubscribes.
+  for (int i = 0; i < 1000; i++) {
+    std::vector<std::string> result;
+    Subscribe(channel, id, result);
```

Comment:
Subscribe is a sync function, we can use pub_sub_->Subscribe directly.

Reply:
Makes sense, thanks!